src/content/config.ts

```diff
@@ -1,27 +1,33 @@
 import { z, defineCollection } from "astro:content";
 
+export const postSchema = z.object({
+  title: z.string().max(50),
+  description: z.string().max(280),
+  author: z.string().max(50),
+  image: z.string().min(12),
+  type: z.enum(["Thoughts", "Announcement", "Hidden"]),
+  date: z.date(),
+});
+export type Post = z.infer<typeof postSchema>;
+
 const posts = defineCollection({
   type: "content",
-  schema: z.object({
-    title: z.string().max(50),
-    description: z.string().max(280),
-    author: z.string().max(50),
-    image: z.string().min(12),
-    type: z.enum(["Thoughts", "Announcement", "Hidden"]),
-    date: z.date(),
-  }),
+  schema: postSchema,
+});
+
+export const newsletterSchema = z.object({
+  description: z.string().max(280),
```

Comment:
newsletters should have a title of "This Month in Ladybird - Month Year"

Reply:
We generate the title from the newsletter date, so it's always `This Month in Ladybird - <date>`. However, I'm not sure if we should just move this to always come from the newsletter itself. Not sure if there is an interest in having a different format for newsletter titles.

Reply:
Updated to show date based on the newsletter title